cfe_internal/update/update_policy.cf

```diff
@@ -597,4 +610,64 @@ body delete u_tidy
       rmdirs   => "true";
 }
 
+bundle agent install_modules
+{
```

Comment:
Can you please add a `# @breif description` before this line?

Reply:
done